### PR TITLE
Updating picklist

### PR DESCRIPTION
**Pull request by amitcha06**

### CODE REVIEW

File: lib/views/pc/picklist/fetch_picklist.dart [hunk 1]
```diff
@@ -64,84 +75,94 @@ Stream<List<PickListTeam>> fetchPicklist() {
...
-          )
-          .length,
-      avgBalls: (avg["tele_upper"] as double? ?? double.nan) +
-          (avg["tele_lower"] as double? ?? double.nan) +
-          (avg["auto_upper"] as double? ?? double.nan) +
-          (avg["auto_lower"] as double? ?? double.nan),
-      autoBallAvg: (avg["auto_upper"] as double? ?? double.nan) +
-          (avg["auto_lower"] as double? ?? double.nan),
+      drivetrain: team["_2023_pit"]?["drivetrain"]["title"] as String?,
+      matchesBalanced:
```
Comment (ItayNaumann) [inline lib/views/pc/picklist/fetch_picklist.dart:119]: i dont think about a better name for this but from the name it sounds like only the matches where the team was balanced, not including unbalanced...

**Review by ItayNaumann — CHANGES_REQUESTED** [review]
we have the commits from #390 ... cherry-pick or just rebase push -f

**Comment by ItayNaumann** [comment]
accidently merged into current master, could be reversible very easily, mb......

**Comment by ItayNaumann** [comment]
i merged the 2 last commits into 1 and removed all the commits that master already had

**Review by ItayNaumann — DISMISSED** [review]
Just did the push -f, looks good to me and there are no longer the main's commits

**Review by OmerPrag — APPROVED** [review]

**Review by OmerPrag — APPROVED** [review]

**Review by Ido-Sobol — APPROVED** [review]

**Pull request merged**